softwareconfiguration/mlplan/src/de/upb/crc901/mlpipeline_evaluation/PerformanceDBAdapter.java

```diff
@@ -0,0 +1,184 @@
+package de.upb.crc901.mlpipeline_evaluation;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.xml.bind.annotation.adapters.HexBinaryAdapter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import hasco.model.ComponentInstance;
+import jaicore.basic.SQLAdapter;
+import jaicore.ml.cache.ReproducibleInstances;
+
+/**
+ * Database adapter for performance data
+ * 
+ * @author jmhansel
+ *
+ */
+public class PerformanceDBAdapter implements Closeable {	/** Logger for controlled output. */
+	static final Logger logger = LoggerFactory.getLogger(CacheEvaluatorMeasureBridge.class);
```

Comment:
visibility modifier?

Reply:
Apart from that: Are you sure you want the logger for the "CacheEvaluatorMeasureBridge" here?